controllers/secretscope_controller_test.go

```diff
@@ -43,11 +46,33 @@ var _ = Describe("SecretScope Controller", func() {
 		keys := []string{aclKeyName, secretsKeyName}
 		for _, value := range keys {
 			_ = apiClient.Secrets().DeleteSecretScope(value)
+
+			ss := &databricksv1alpha1.SecretScope{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      value,
+					Namespace: "default",
+				},
+			}
+
+			_ = k8sClient.Delete(context.Background(), ss)
```

Comment:
Should this err be checked?

Reply:
I think it should be find.  Both `aclKeyName` and `secretsKeyName`'s names are randomly generated. `_ = k8sClient.Delete(context.Background(), ss)` is just to ensure each `SecretScope` has been fully deleted from the cluster between each unit test.  If for whatever reason `_ = k8sClient.Delete(context.Background(), ss)` fails, the tests will fail anyway.